ledger-core/v2/insolar/payload/payload.proto

```diff
@@ -3,14 +3,21 @@ syntax = "proto3";
 package payload;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/insolar/assured-ledger/ledger-core/v2/insproto/ins.proto";
+
+option (insproto.notation_all) = true;
+option (insproto.zeroable_all) = true;
```

Comment:
maybe I'm wrong, but we have couple of fields with are zeroable, all the rest should be required by default. Maybe we can turn this to false and explicitly say which fields are zeroable?

Reply:
Make sense, I will check it. Thanks!

Reply:
debatable ... if a field = 0, the why should we send bytes, denoting this field? receiver will do logical checks anyway

Reply:
hmmm, if it is kind of json "omitempty", then disregard my comment. In custom types it was  making *FieldType instead of FieldType (to be able to handle empty). I was thinking about that case.

Reply:
well, if a field type has IsZero() method and zero value means missing value - like for references, then you don't need nullable as well. Just a matter of view point.